build.cmd

```diff
@@ -484,8 +494,8 @@ if /i "%__DoCrossArchBuild%"=="1" (
     set __MsbuildLog=/flp:Verbosity=normal;LogFile=!__BuildLog!
     set __MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!
     set __MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!
-
-    @call %__ProjectDir%\run.cmd build -Project=%__CrossCompIntermediatesDir%\install.vcxproj -configuration=%__BuildType% -platform=%__CrossArch% -MsBuildLog=!__MsbuildLog! -MsBuildWrn=!__MsbuildWrn! -MsBuildErr=!__MsbuildErr! %__RunArgs% %__UnprocessedBuildArgs%
+    
```

Comment:
nit: unnecessary whitespace

Reply:
applied spacing convention from native build part where between log variables and build call is a space introduced intentionally  for clarity of code. Should I fix it in both places or leave them as they are now?

Reply:
the newline is fine, I was pointing out the four whitespaces on the (otherwise empty) line 😄

Reply:
Yeah, that was it 😄